ApiExtractor/abstractmetabuilder.cpp

```diff
@@ -34,20 +34,20 @@
 #include "parser/parser.h"
 #include "parser/tokens.h"
 
-#include <QDebug>
-#include <QFile>
-#include <QFileInfo>
-#include <QTextCodec>
-#include <QTextStream>
-#include <QVariant>
-#include <QTime>
-#include <QQueue>
-#include <QDir>
+#include <QtCore/QDebug>
```

Comment:
Pedantic: using tll(Qt5::Core), I don't believe this is necessary... if supporting both Qt4/Qt5 is a goal, it might be better to omit these changes.